source/jormungandr/jormungandr/scenarios/tests/journey_compare_tests.py

```diff
@@ -877,16 +876,75 @@ def test_heavy_journey_bike():
     journey.sections[-1].street_network.mode = response_pb2.Bike
     journey.durations.bike = journey.sections[-1].duration = 15
 
-    journey_filter.filter_too_short_heavy_journeys(journey, min_bike=10, min_car=20)
-
-    assert 'to_delete' not in journey.tags
+    f, _ = jf.make_filter_too_short_heavy_journeys(min_bike=10, min_car=20)
+    assert f(journey)
 
     journey.durations.bike = journey.sections[-1].duration = 5
 
-    journey_filter.filter_too_short_heavy_journeys(journey, min_bike=10, min_car=20,
-                                                   orig_modes=['bike', 'walking'])
+    f, _ = jf.make_filter_too_short_heavy_journeys(min_bike=10, min_car=20, orig_modes=['bike', 'walking'])
+    assert not f(journey)
+
+def test_filter_wrapper_on_heavy_journey_bike():
+    """
+    same test as above, but this time we wrap the filter to test wrapping is ok
+    """
+    ref_journey = response_pb2.Journey()
+    ref_journey.sections.add()
+    ref_journey.sections[-1].type = response_pb2.STREET_NETWORK
+    ref_journey.sections[-1].street_network.mode = response_pb2.Bike
+    ref_journey.durations.bike = ref_journey.sections[-1].duration = 15
+
+    # first we test when debug-mode deactivated
+    j = deepcopy(ref_journey)
+
+    wrapped_f = jf.filter_wrapper(is_debug=False,
+            f_msg_pair=jf.make_filter_too_short_heavy_journeys(min_bike=10, min_car=20))
```

Comment:
If you intend to only test the wrapper function, you could mock the wrapped filter to simplify the code. It would also simplify the test coverage as you already test `filter_too_short_heavy_journeys` later in the tests.